src/extensions/utils.ts

```diff
@@ -55,3 +55,13 @@ export function getRandomString(length: number): string {
   }
   return result;
 }
+
+/**
+ * Formats a timestamp from the Extension backend into something more readable
+ * @param timestamp with this format: 2020-05-11T03:45:13.583677Z
+ * returns a timestamp with this format: 2020-05-11 T03:45:13
+ */
+export function formatTimestamp(timestamp: string): string {
```

Comment:
This should probably handle being passed `null` / `undefined` -- right now it'll throw an error. It could return an empty string and then you wouldn't need the conditional above.

Reply:
Slightly different question: why _isn't_ this using the native `Date`? Doing `new Date(ts)` and formatting from there?

Reply:
I initially thought about that too. But the native Date library doesn't have proper string formatting built in anyway. So I have to write my own formatter using date.GetFullYear(), date.GetMonth(), date.GetDay(), etc. On top of that, you have to fix issues like the month being zero indexed. 

Overall it seems like the code would be more convoluted if I had to write my own formatter and introduce more risk in this codebase than necessary, which is why I initially chose to use moment.js. But since we can't use moment.js, I felt the next best solution was to parse the string, since we know the expected format anyway.

Reply:
+1 Jeff, Bryan you and I disagreed about the usefulness of `Date` coercion in a different PR so I'll let you push if you want it different (seems fine and straightforward to me).

Reply:
... I really like moment because you can just use it and it generally has a better API. Removing it is fine. Writing your own formatter is fine. We don't have any "here's how dates should be formatted" guidance right now in the CLI. We have enough of them these days that maybe it'd be worth it. It's probably not worth the attention I'm giving it.

But Dates in javascript are generally awful, IMO. `¯\_(ツ)_/¯`